src/wireguard.sh

```diff
@@ -12,7 +12,6 @@ PrivateKey=$2
 WireguardPort=$3
 PeersFile=$4
 
-UdpPunchIp="0.0.0.0"
 WireguardConfigFile="/etc/wireguard/wg0.conf"
```

Comment:
As discussed offline - let's put this in `/tmp` since `/etc/wireguard` isn't guaranteed to exist for local execution.

Reply:
Fixed